cup_of_join.py

```diff
@@ -0,0 +1,14 @@
+# Arguments :
+# args - Undefined number of lists
+# sep - random char
+# return value : list
+# This functions joins lists and between them appends the given char 'sep'
+def join(*args, sep='-'):
+    if not args:
```

Comment:
will it ever be none? use ipython to make sure - dont assume you know what args empty value is 
![image](https://user-images.githubusercontent.com/13467749/159860004-921f887c-df7c-4fd7-b257-dd770cddeb75.png)